src/main/java/com/neighborhoodAssurance/NAController.java

```diff
@@ -17,14 +17,16 @@ public String index()
 		return "index";
 		
 	}
+	
+	/**
+	 * Reads JSON
+	 * @return Returns JSON object
+	 */

```

Comment:
Cleaned up your JavaDoc and moved it. The other one in this file looks right! It's possible to create a linter that will enforce coding conventions such as requiring well structured JavaDocs.

Reply:
Above a method is the proper place for JavaDoc.  Then again, this method is fairly self-describing.